lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -288,8 +285,9 @@ const expectations = require(resolveLocalOrCwd(cli['expectations-path']));
 let passingCount = 0;
 let failingCount = 0;
 expectations.forEach(expected => {
-  console.log(`Checking '${expected.initialUrl}'...`);
+  console.log(`Running '${expected.initialUrl}'...`);
   const results = runLighthouse(expected.initialUrl, configPath);
+  console.log(`Comparing results of '${expected.initialUrl}':`);
```

Comment:
maybe ``console.log(`Auditing findings of '${expected.initialUrl}':`);``?

Reply:
😱